__tests__/cmds/versions.test.js

```diff
@@ -52,7 +54,9 @@ describe('rdme versions*', () => {
         .basicAuth({ user: key })
         .reply(200, [versionPayload, version2Payload]);
 
-      await expect(versions.run({ key })).resolves.toMatchSnapshot();
+      const table = await versions.run({ key });
```

Comment:
Reverted this back to using these two assertions because the `FORCE_COLOR` env variable doesn't seem to always get used between `colors` and `chalk`.